### DISCUSSION TITLE
Proposal to Lower Aptos Governor Limit

### DISCUSSION THREAD

**Discussion by claudijd** [wormhole-foundation/wormhole#3580] [Governance] [answered]
Currently, the Aptos governor limit is set to $5M/day, however after an analysis of actual usage against this limit the usage numbers no longer support the limit being that high.

Aptos token bridge transfers over the past 31 days between sequences 41071 and 45318:

Notional usd average : $117,669.49
Notional usd total   : $3,647,754.17

Aptos token bridge transfers over the past 181 days between sequences 18375 and 45326:

Notional usd average : $131,675.08
Notional usd total   : $23,833,188.88

I'm proposing that Guardians consider drawing down the Governor limit on Aptos from $5M to $1M as a conservative step down.

Here's also a PR that would enact this proposal should it receive support:

https://github.com/wormhole-foundation/wormhole/pull/3581

**Comment by evan-gray** [comment, marked as answer]
Concur, I crunched similar stats.

**Comment by aadam-10** [comment]
live as of [v2.23.28](https://github.com/wormhole-foundation/wormhole/releases/tag/v2.23.28)
